lib/cfnguardian/resources/application_targetgroup.rb

```diff
@@ -23,9 +23,12 @@ def default_alarms
       alarm = CfnGuardian::Models::ApplicationTargetGroupAlarm.new(@resource)
       alarm.name = 'TargetResponseTime'
       alarm.metric_name = 'TargetResponseTime'
-      alarm.threshold = 5
+      alarm.threshold = 30
       alarm.evaluation_periods = 5
-      alarm.treat_missing_data = 'notBreaching'
+      alarm.period = 60
+      alarm.unit - 'Seconds'
+      alarm.alarm_action = 'Warning'
+      alarm.treat_missing_data = 'missing'
```

Comment:
missing is the default value if omitted
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cw-alarm.html#cfn-cloudwatch-alarms-treatmissingdata